frontend/src/app/core/helpers/questions-mandatory-helper.ts

```diff
@@ -1,6 +1,20 @@
 import { IQuestion } from '@core/models/IQuestion';
 
 export const getLocalMandatoryQuestions = (): IQuestion[] => [
-    { id: 0, order: 0, questionText: 'Name', isMandatory: true },
-    { id: 0, order: 0, questionText: 'Email', isMandatory: true },
+    {
+        id: 0,
+        order: 0,
+        questionText: 'Name and Surname *',
+        placeHolderText: 'Name Surname',
+        checkboxText: 'Autofill Invitee Name, Email',
```

Comment:
The `checkBoxText:` seems odd inside the question. Make it a component variable instead